packages/eas-cli/src/submit/ArchiveSource.ts

```diff
@@ -317,30 +317,45 @@ async function handleBuildListSourceAsync(
 }
 
 function formatBuildChoice(build: BuildFragment, expiryDate: Date): prompts.Choice {
-  const { id, updatedAt, runtimeVersion, buildProfile, gitCommitHash, gitCommitMessage, channel } =
-    build;
-
-  const formatValue = (field?: string | null): string => (field ? chalk.bold(field) : 'Unknown');
-
+  const {
+    id,
+    updatedAt,
+    runtimeVersion,
+    buildProfile,
+    gitCommitHash,
+    gitCommitMessage,
+    channel,
+    message,
+  } = build;
   const buildDate = new Date(updatedAt);
 
+  const splitCommitMessage = gitCommitMessage?.split('\n');
   const formattedCommitData =
-    gitCommitHash && gitCommitMessage
-      ? `${gitCommitHash.slice(0, 7)} "${chalk.bold(gitCommitMessage)}"`
-      : 'Unknown';
+    gitCommitHash && splitCommitMessage && splitCommitMessage.length > 0
+      ? `${gitCommitHash.slice(0, 7)} "${chalk.bold(
+          splitCommitMessage[0] + (splitCommitMessage.length > 1 ? '…' : '')
+        )}"`
+      : null;
 
   const title = `${chalk.bold(`ID:`)} ${id} (${chalk.bold(`${fromNow(buildDate)} ago`)})`;
 
-  const description = [
-    `\t${chalk.bold(`Profile:`)} ${formatValue(buildProfile)}`,
-    `\t${chalk.bold(`Channel:`)} ${formatValue(channel)}`,
-    `\t${chalk.bold(`Runtime version:`)} ${formatValue(runtimeVersion)}`,
-    `\t${chalk.bold(`Commit:`)} ${formattedCommitData}`,
-  ].join('\n');
+  const descriptionItemNameToValue: Record<string, string | null> = {
```

Comment:
It will work, but I think it's a bad practice to relay on the fact that js guarantees keys order in an object.